compiler/circle-quantizer/src/CircleQuantizer.cpp

```diff
@@ -298,6 +308,19 @@ int entry(int argc, char **argv)
     }
   }
 
+  if (arser[qw])
+  {
+    auto values = arser.get<std::vector<std::string>>(qw);
+    if (values.size() != 3)
+    {
+      std::cerr << arser;
+      return 255;
+    }
+    // TODO: Add options and param
```

Comment:
@glistening Are you going to support mixed-precision quantization too? Ex: some layers are quantized in uint8 and others are int16. If so, you may need to receive quantization configuration file too.

Reply:
@jinevening  In short, no.

As I wrote in #11057, I am working this option for `onert` requirement.
`onert` does not have plan about mixed quantization for hybrid kernel.
(It is the reason I don't handle configuration option at all.)

Also, only one of {`channel` or `layer`} will be supported as `onert` supports 1 kind of kernels.